sdcm/cluster.py

```diff
@@ -2022,9 +2022,8 @@ def install_scylla(self, scylla_repo):
             self.remoter.run('sudo apt-get install -y rsync')
             self.download_scylla_repo(scylla_repo)
             self.remoter.run('sudo apt-get update')
-            self.remoter.run(
-                'sudo apt-get install -y '
-                ' {} '.format(self.scylla_pkg()))
+            self.install_package('apt-get install -y openjdk-11-jre')
```

Comment:
@fgelcer you have apt-get twice here ? i.e. on inside install_package, and one on the argument here.

Reply:
yep, that was a mistake, and it was already changed